cli/src/main/java/com/box/l10n/mojito/cli/command/RepoUpdateCommand.java

```diff
@@ -46,6 +46,9 @@
     @Parameter(names = {Param.REPOSITORY_DESCRIPTION_LONG, Param.REPOSITORY_DESCRIPTION_SHORT}, arity = 1, required = false, description = Param.REPOSITORY_DESCRIPTION_DESCRIPTION)
     String descriptionParam;
 
+    @Parameter(names = {Param.CHECK_SLA_LONG, Param.CHECK_SLA_SHORT}, arity = 1, required = false, description = Param.CHECK_SLA_DESCRIPTION)
+    String checkSLA;
```

Comment:
same

Reply:
use Boolean for checkSLA repoUpdate command.